formats/developers/index.txt

```diff
@@ -60,6 +60,11 @@ Support for additional dimensions is also covered:
 
 -  :doc:`6D, 7D and 8D Storage <6d-7d-and-8d-storage>`
 
+The new 'key-value pairs' Map Annotations introduced in the
```

Comment:
I wonder if it is worth dropping 'new' now so that we don't need to later? That they are introduced in the Jan 2015 schema may be enough to express the newness?

Could use of 'key-value pairs' as an adjective be avoided? Something like:

`Map Annotations, using key-value pairs, introduced in the...`